run.sh

```diff
@@ -170,160 +160,69 @@ init() {
     done
 
     # If no theme dependencies or volumes exist, fall back to default
-    if [[ ${#theme_deps[@]} == 0 && $(check_volumes -t) == "" ]]; then
-        theme_deps[twentytwenty]=twentytwenty
+    if [[ ${#theme_deps[@]} == 0 && -d /app/wp-content/themes && $(find /app/wp-content/themes/* -maxdepth 0 -t d | wc -l) == 0 ]]; then
+        theme_deps["$default_theme"]="$default_theme"
     fi
 
+    # Apache config adustments
+    sudo sed -i \
+        -e "/^[[:blank:]]*.ServerName www.example.com/{c\\" \
+        -e "\\tServerName ${SERVER_NAME:-localhost} \\" \
+        -e "\\tServerAlias www.${SERVER_NAME:-localhost}" \
+        -e '}' /etc/apache2/sites-available/000-default.conf
+
     sudo chown -R admin:admin /app
 
     if [[ ! -f /app/wp-settings.php ]]; then
         h2 'Downloading WordPress'
-        wp --color core download |& logger
+        wp core download |& logger
     fi
 }
 
 check_database() {
-    wp core is-installed 2> /dev/null && return
+    declare file
+    declare -i num_imported=0
 
-    wp --color db create |& logger
+    wp core is-installed 2> /dev/null && return
 
-    declare data_path
-    data_path=$(find /data -name '*.sql' -print -quit 2> /dev/null)
-    if [[ ! "$data_path" ]]; then
-        wp --color core install |& logger
-        return
-    fi
+    wp db create |& logger
 
-    wp --color db import "$data_path" |& logger
+    for file in /data/*.sql; do
```

Comment:
Allows users to import more than one database file (in the event they download chunks of a single installation) -- I'm running into the need for this so I figured I'd add it in.